js/index.js

```diff
@@ -39,4 +39,79 @@ const siteContent = {
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// const imghead = document.querySelector("logo.png");
+// imghead.setAttribute('src', "img/logo.png");
+
+let headimg = document.getElementById("cta-img");
+headimg.setAttribute( "src", siteContent["cta"]["img-src"]);
+
+// const heading = document.querySelector("cta-img");
+// headimg.setAttribute('src',"cta-img");
+
+let navo = document.querySelectorAll("a");
+navo.forEach(function(item, index){
+item.textContent = siteContent["nav"][`nav-item-${index + 1}`];
+
+});
```

Comment:
well written here but be sure you know how to refactor to ES6 arrow syntax as that will be used a lot more commonly as we go through the curriculum. I'd also reconsider the name `navo` for a list of links/anchor tags. make it semantic so if you see it somewhere later on you know exactly what it is!